test/benchee/formatters/html_test.exs

```diff
@@ -182,6 +182,29 @@ defmodule Benchee.Formatters.HTMLTest do
     end
   end
 
+  test "assets should not be copied when assets inlining is on" do
+    try do
+      suite = %Benchee.Suite{
+                scenarios: [@scenario],
+                system: @system_info,
+                configuration: %Benchee.Configuration{
+                  formatter_options: %{html: %{file: @filename, auto_open: false, inline_assets: true}}
+                }
+              }
+
+      capture_io fn ->
+        return = Benchee.Formatters.HTML.output(suite)
+        assert return == suite
+      end
+
+      assert File.exists? @expected_filename
```

Comment:
testing that the tags etc. are in the html would be nice I think :)

Reply:
yep